docs/blog/2019-11-19-gatsby-brings-speed-to-mediacurrent-com/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Gatsby Brings Speed and Simplicity to Mediacurrent.com
+date: 2019-11-19
```

Comment:
Does the date need to be updated? If so both this and directory name for this file needs bumping

Reply:
created PR for date update #20051